### PR TITLE
18983 view dependents child schema

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request adds remove child workflows to the view dependents page

## Testing done
- local

## Screenshots
<details><summary>Workflows</summary>

Screen Shot 2021-02-22 at 2 58 59 PM
Screen Shot 2021-02-22 at 2 59 15 PM
Screen Shot 2021-02-22 at 2 59 50 PM

</details>

## Acceptance criteria
- [x] schemas have been added

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx [hunk 1]
```diff
@@ -20,6 +28,17 @@ function ViewDependentsListItem(props) {
     stateKey,
   } = props;
 
+  const handleClick = () => {
+    setOpen(prevState => !prevState);
+    if (open) {
+      const focusEl = document?.querySelectorAll('.mng-dependents-name')?.[
```
Comment (cohnjesse) [inline src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx:34]: Do we want the name set as the focus element or the first form field?

File: src/applications/personalization/view-dependents/manage-dependents/schemas.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { countries, states50AndDC } from 'vets-json-schema/dist/constants.json';
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import { addressUiSchema } from 'applications/vre/definitions/profileAddress';
 
 const PATTERNS = {
   date: '^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$',
```
Comment (cohnjesse) [inline src/applications/personalization/view-dependents/manage-dependents/schemas.js:6]: It seems like there should really be reusable patterns like this somewhere in a constants file in vets-website :-(

**Review by cohnjesse — APPROVED** [review]
LGTM

File: src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx (hunk 1 above)
Comment (micahchiang) [reply src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx:34]: It's a little weird, but this activates when the form closes, so having focus on the name makes a little more sense to me.

File: src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx [hunk 3]
```diff
@@ -29,7 +48,11 @@ function ViewDependentsListItem(props) {
           'vads-u-margin-bottom--2': manageDependentsToggle === false,
         })}
       >
-        <dt className="vads-l-col--12 vads-u-margin--0 vads-u-font-size--lg vads-u-font-weight--bold">
+        <dt
+          // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```
[the remark this answers is not in the document]
Comment (timwright12) [reply src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx:52]: @micahchiang we should set this in the JS as `tabindex="-1"` so it doesn't become a real tab-stop for the user

[the remark this answers is not in the document]
Comment (micahchiang) [reply src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx:52]: @timwright12 - good catch, thanks!

**Review by timwright12 — APPROVED** [review]

File: src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx:34]: gotcha

**Pull request merged**